.circleci/config.yml

```diff
@@ -245,7 +245,7 @@ workflows:
       - integration-tests:
           filters:
             tags:
-              only: /.*/
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+.*/
```

Comment:
Can you clarify why the change in the PR description? I am not fluent with CircleCI config to understand what this fixes.

Reply:
Done.